dev/NavigationView/NavigationView.h

```diff
@@ -75,22 +75,81 @@ class NavigationView :
 
     void CoerceToGreaterThanZero(double& value);
 
-    void OnNavigationViewItemInvoked(const winrt::NavigationViewItem& nvi);
+    void RepeaterElementPrepared(const winrt::ItemsRepeater& ir, const winrt::ItemsRepeaterElementPreparedEventArgs& args);
```

Comment:
nit: OnRepeaterElementPrepared. It looks like we are following the OnFooHappened pattern for other event handlers, so would be nice to be consistent.